client/src/components/TodayScheduleTable/index.js

```diff
@@ -0,0 +1,367 @@
+import React, { useEffect, useState } from 'react';
+import 'antd/dist/antd.css';
+import './style.css';
+// import PropTypes from 'prop-types';
+import {
+  Table,
+  Popconfirm,
+  Form,
+  DatePicker,
+  TimePicker,
+  Checkbox,
+  Alert,
```

Comment:
There's a common component called `AlertMessage` you should use instead

Reply:
done